docs/products/known_issues_and_subtleties.rst

```diff
@@ -16,6 +16,45 @@ We are currently integrating a major change to our deblender that we hope will m
 
 It is likely that our direct image processing in crowded fields will still lag dedicated crowded-field photometry codes; Rubin's focus for these fields has always been image subtraction (where we do expect to compete with the state of the art), with the :ref:`catalogs-object` catalog a best-effort addition.
 
+.. _products_wcs_known_issues:
+
+WCS FITS approximations and misleading interfaces
+=================================================
+
+Rubin's single-visit World Coordinate System (WCS) objects are not in general exactly representable via the FITS WCS standard, and hence the FITS WCS in the headers of the :ref:`visit image <images-visit-image>` and :ref:`difference image <images-difference-image>` data products (they are the same) are approximations that are only expected to be good enough for visualization and object-finding, not precision astrometry.
```

Comment:
"are approximations": how good are these approximations, at least roughly? Half a pixel? And are they approximations to what, the full astrometric solution? I assume so, but maybe it's better to say it explicitly. And then, is it possible for a user to find information / documentation about the full astrometric solution objects?

Reply:
OK, I see below "Unfortunately, these objects are not included in our Python API documentation reference at present, and they are counterintuitive and easy to misuse in several ways, which can make it appear as if the WCS fits are terrible."